test/cmake/include_dependencies/add_include_dependencies.cmake

```diff
@@ -0,0 +1,119 @@
+# -----------------------------------------------------------------------------------------------------
+# Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+# Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+# This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+# shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+# -----------------------------------------------------------------------------------------------------
+
+# get_include_target (<VAR> TARGET dna4_test)
+# get_include_target (<VAR> SOURCE "[test/unit/]alphabet/nucleotide/dna4_test.cpp")
+# get_include_target (<VAR> HEADER "[include/seqan3/]alphabet/nucleotide/dna4.hpp")
+function (get_include_target VAR method source)
```

Comment:
Which name?

Reply:
slight polite ping @rrahn

Reply:
From the documentation I read: `method` in `{TARGET, SOURCE, HEADER}`. But it seems that method is not the appropriate name for this. So I am wondering if there is a better name for this group of things. It can be a binary, a cpp file and a header file, so naturally it is ... a thing :)

Reply:
maybe `source_method`, `source_type`, `cmake_target_source_file_or_header_file_keyword` ?

Reply:
the last one :) I would go for the `source_type`.

Reply:
done